lib/thesis/views/api_view.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Thesis.ApiView do
+  use Thesis.View, :view
+
+
+  def render("page.json", %{thesis_page: page} = assigns) do
+    %{
+      id: page.id,
+      slug: page.slug,
+      title: page.title,
+      description: page.description,
+      redirect_url: page.redirect_url,
+      page_contents: render("page_contents.json", assigns[:page_contents]),
```

Comment:
Note that `page_contents` will always be an empty list here, as we aren't assigning `:page_contents` in the controller -- this is only here for possible future need.